src/js/playlist/loader.js

```diff
@@ -50,20 +53,22 @@ const PlaylistLoader = function() {
                         throw Error;
                     }
                 } catch (e) {
-                    throw new Error('Not a valid RSS/JSON feed');
+                    throw new PlayerError('Not a valid RSS/JSON feed', 21, e);
```

Comment:
the code should be `621` 
https://jwplayer.atlassian.net/wiki/spaces/ES/pages/231637123/Documented+Errors+list